### PR TITLE
Added the version to the CLI

### PR DESCRIPTION
**Pull request by maikefischer**
I added the version number from #74

### CODE REVIEW

File: fast_carpenter/__main__.py [hunk 1]
```diff
@@ -56,13 +57,17 @@ def __call__(self, parser, namespace, values, option_string=None):
                         help="Print help specific to the available stages")
     parser.add_argument("--help-stages-full", action=StagesHelp, metavar="stage",
                         help="Print the full help specific to the available stages")
+    parser.add_argument("-v", "--version", action="version", version='%(prog)s ' + __version__)
```
Comment (benkrikler) [inline fast_carpenter/__main__.py]: I think you're missing a single space after the `+` here.

File: fast_carpenter/__main__.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 atup.EventBuilder = EventBuilder
 atup.build_parallel = build_parallel
 logging.getLogger(__name__).setLevel(logging.INFO)
+from .version import __version__
```
Comment (benkrikler) [inline fast_carpenter/__main__.py]: To comply with PEP8, this import should be 3 lines up next to the other imports.

Comment (benkrikler) [reply fast_carpenter/__main__.py]: https://travis-ci.com/FAST-HEP/fast-carpenter/jobs/241954792

**Comment by maikefischer** [comment]
@benkrikler all done!

**Comment by benkrikler** [comment]
Thank you!

**Pull request merged**